deps/build.jl

```diff
@@ -34,6 +34,10 @@ if is_apple()
     provides(Homebrew.HB, "homebrew/science/hdf5", hdf5, os=:Darwin)
 end
 
+if Sys.KERNEL === :FreeBSD
```

Comment:
Oh right. I'm guessing it will probably make sense to  introduce a `Sys.isfreebsd()` function at some point after your PR.